styles/style.css

```diff
@@ -23,7 +27,7 @@ header {
     display: flex;
     align-items: center;
     justify-content: space-between;
-    max-width: 1024px;
+    max-width: 1160px;
```

Comment:
It's better to apply sizing styles to div wrapper inside body rather to body itself